tests/contrib/test_multiprocess_file_storage.py

```diff
@@ -0,0 +1,293 @@
+# Copyright 2015 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Unit tests for oauth2client.multistore_file."""
+
+import contextlib
+import datetime
+import json
+import multiprocessing
+import os
+import tempfile
+import unittest2
+import fasteners
+import mock
+
+from oauth2client.client import OAuth2Credentials
+from oauth2client.contrib import multiprocess_file_storage
+from six import StringIO
+
+from ..http_mock import HttpMockSequence
```

Comment:
@dhermes: do we have a project-wide policy on this style of import? I'm not even familiar with it.